projects/kit/components/progress/progress-bar/progress-bar.component.less

```diff
@@ -41,62 +41,43 @@
 
     .clearProgress();
     .progressAnimation();
-    .progressValue({ background: @progress-color });
+    .progressValue({ background: @progress-color; border-radius: inherit });
 
     display: block;
     width: 100%;
     height: var(--t-height);
     color: var(--tui-primary);
     background: @track-color;
-    clip-path: inset(0 round var(--tui-radius-xs));
+    clip-path: inset(0 0.5px round var(--tui-radius-m)); // 0.5px is a hack to prevent jagged edges on low dpi screens
     overflow: hidden;
+    border-radius: 1rem;
 
     &[data-mode='onDark'] {
         --tui-clear: var(--tui-clear-inverse);
     }
 
-    &[data-size='m'] {
-        --t-height: 1.5rem;
+    &[data-size='xs'] {
+        --t-height: 0.25rem;
     }
 
     &[data-size='s'] {
-        .progressValue({ border-radius: var(--tui-radius-xs) });
-
         --t-height: 0.5rem;
     }
 
-    // TODO: drop new attribute in v4.0 and make all these styles as default ones
-    &[new] {
```

Comment:
Update this documentation page

https://taiga-ui--pr7092-progress-move-jtxz5ldx.web.app/components/progress-bar#sizes

Reply:
And this example

https://taiga-ui--pr7092-progress-move-jtxz5ldx.web.app/components/progress-bar#customizable-corners

Reply:
Let's use bigger size for this example:

https://taiga-ui--pr7092-progress-move-jtxz5ldx.web.app/components/progress-bar#label